helper_files/settings.py

```diff
@@ -0,0 +1,10 @@
+# TODO: Come up with better name for this file
+# Use this hold globals and other things need across the cogs
+# that cannot be initialized for whatever reason in the cogs themselves
+
+# To be initialed from main in the on_ready function
+# Used in almost all the cogs for author name and author avatar in embeds
+global BOT_NAME

```

Comment:
@TitanVJ  why have a separate file for these instead of just putting it in `main.py`?

Reply:
i dont like the idea of importing main. also file could be useful for storing other things of similar nature in the future, would prevent main from getting cluttered

Reply:
@TitanVJ is the intention of the file that it will store all variables that get initialized/used in main.py?

Reply:
yeah i guess, but also for anything thats used in a cog that cant be initialized in the cog.

Reply:
ok.